instrumentation/log4j/log4j-context-data/log4j-context-data-2.17/javaagent/build.gradle.kts

```diff
@@ -66,6 +66,29 @@ testing {
         }
       }
     }
+
+    val testLoggingKeys by registering(JvmTestSuite::class) {
+      sources {
+        java {
+          setSrcDirs(listOf("src/testLoggingKeys/java"))
+        }
+      }
```

Comment:
I think you don't need to add this. Src dir should be added automatically based on the suite name.

Reply:
> I think you don't need to add this. Src dir should be added automatically based on the suite name.

Because I found that `testAddBagge` and `testDisableThreadLocals`  have written, Need to remove them together?

Reply:
I think it is needed for `testDisableThreadLocals` but not for `testAddBagge`

Reply:
Isn't the structure of `testAddBagge` and `testLoggingKeys` the same. I don't quite understand this.

Reply:
Fixed. Thank you for your help, I will further study Gradle.

Reply:
You removed the `sources` from the wrong suite. You can remove sources from `testAddBagge` as it will by default use `src/testAddBaggage/java` so no need to set that manually. You can't remove source from `testDisableThreadLocals` because the default source does not match with what is set manually.